lib/runner/variant-reassignment.js

```diff
@@ -51,79 +73,96 @@ export default class VariantReassignment {
       = this._selectProductDraftsForReassignment(productDrafts, products)
 
     this.logger.debug(
-      'Filtered %d productDrafts for reassignment',
-      productDraftsForReassignment.length
+      `Filtered ${productDraftsForReassignment.length} productDrafts for reassignment`
     )
 
-    for (const productDraft of productDraftsForReassignment)
+    for (const productDraft of productDraftsForReassignment) {
+      const skus = this.productService.getProductDraftSkus(productDraft)
       try {
+        this.statistics.processed++
+        this.statistics.processedSkus.push(...skus)
         await this._processProductDraft(productDraft, products)
+        this.statistics.succeeded++
```

Comment:
shouldn't we somehow rename these statistic properties?

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/48/commits/5863254c69c27014b9dd054ab25789745b3081a6

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/48/commits/1a40f51bc9ddb2c3efcd7a9fb00be4314e762091